client/signup/steps/woocommerce-install/transfer/index.tsx

```diff
@@ -52,13 +52,13 @@ export default function Transfer( props: WooCommerceInstallProps ): ReactElement
 				<>
 					{ isAtomic && (
 						<InstallPlugins
-							onFailure={ () => handleTransferFailure( 'install ' ) }
+							onFailure={ ( type ) => handleTransferFailure( type ) }
```

Comment:
Could this be shortened to the following?

```suggestion
							onFailure={ handleTransferFailure }
```

Reply:
Yep that works, thanks!